lab-andrew/route/reminder-route.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const Reminder = require('../model/reminder');
+const response = require('../lib/response');
+const Storage = require('../lib/storage');
+const reminderStorage = new Storage(`${__dirname}/../data`);
+const del = require('del');
+
+const reminderPool = {};
+
+module.exports = function(router) {
+  router.post('/api/reminder', function(req, res) {
+    try {
+      if (req.body.content){
+        const reminder = new Reminder(req.body.content);
+        // reminderPool[reminder.id] = reminder;
+        reminderStorage.setItem('notes', reminder)
+      .then(function() {
+        return response(200, reminder)(res);
+      }).catch(function(err) {
+        console.error('error on api/reminder POST', err);
+        response(500, 'server error')(res);
+      });
+        return;
+      }
+      response(400, 'bad request')(res);
+    } catch(err) {
+
+      response(400, 'bad request')(res);
+    }
+  })
+
+
+  .put('/api/reminder', function(req, res) {
+    const reminder = reminderPool[req.body.id];
+    if (reminder) {
+      reminder.id = req.body.id || req.id;
```

Comment:
This route was unnecessary for this assignment, but good work implementing a basic version.  You'll want to add an update function to your storage module and refactor this code to use it if you plan to keep this route in your final PR, otherwise remove it.